### PR TITLE
Use Discovery API, Upgrade FCL, Solve Vite Node Issues

### PR DESCRIPTION
**Pull request by chasefleming**
Addresses: 
https://github.com/emerald-dao/float/issues/48
https://github.com/emerald-dao/float/issues/50

### CODE REVIEW

File: vite.config.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
 // vite.config.js
 import { sveltekit } from '@sveltejs/kit/vite';
+import { nodePolyfills } from 'vite-plugin-node-polyfills'
 
 /** @type {import('vite').UserConfig} */
 const config = {
 	plugins: [
```
Comment (gregsantos) [inline vite.config.js:7]: adding the following to config fixes the build issue for me. @muttoni 

```
  define: {
    global: "globalThis",
  },
```

Comment (gregsantos) [reply vite.config.js:7]: @chasefleming

**Comment by chasefleming** [comment]
Any updates on getting this merged?

**Comment by cybercent** [comment]
Would be great if this could be merged so that we could claim FLOATS with any wallet for the next Flow Town Hall.

**Pull request closed without merging**